games-simulation/lincity-ng/lincity-ng-2.9_beta.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils multiprocessing
+
+DESCRIPTION="City simulation game"
+HOMEPAGE="https://github.com/lincity-ng/lincity-ng"
+SRC_URI="https://github.com/lincity-ng/lincity-ng/archive/lincity-ng-${PV/_/-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 BitstreamVera CC-BY-SA-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="virtual/opengl
+	sys-libs/zlib
+	dev-libs/libxml2
+	media-libs/libsdl[sound,joystick,opengl,video]
+	media-libs/sdl-mixer[vorbis]
+	media-libs/sdl-image[png]
+	media-libs/sdl-ttf
+	media-libs/sdl-gfx
+	dev-games/physfs"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	dev-util/ftjam"
+
+S=${WORKDIR}/${PN}-${P/_/-}
+
+src_prepare() {
+	default
+	sed -i "/COPYING COPYING-data.txt COPYING-fonts.txt CREDITS /d" \
+		Jamfile || die
+	./autogen.sh || die
+}
+
+src_compile() {
+	jam -q -dx -j $(makeopts_jobs) || die "jam failed"
+}
+
+src_install() {
+	jam -sDESTDIR="${D}" \
+		-sappdocdir="/usr/share/doc/${PF}" \
+		-sapplicationsdir="/usr/share/applications" \
+		-spixmapsdir="/usr/share/pixmaps" \
+		install \
+		|| die "jam install failed"
```

Comment:
This hasn't taken account of prefixed systems. Normally you would prefix these paths with `${EPREFIX}` and there are probably other arguments you'd need to add in this case but given that this is the install phase rather than the configure phase, you could probably get away with just replacing `${D}` with `${ED}`. Try doing `EPREFIX=/foo ebuild lincity-ng-x.x.ebuild install` and see where the files end up.

Reply:
Thank you! My bad.

Reply:
In this case if I replace `${D}` with `${ED}`, it causes double prefix. If I only prefix the paths with `${EPREFIX}` the EPREFIX'ed install goes successfully.